manifests/osfamily/redhat.pp

```diff
@@ -48,10 +48,10 @@
     }
   }
 
-  $legacy_keyname = 'RPM-GPG-KEY-puppetlabs'
-  $legacy_gpg_path = "/etc/pki/rpm-gpg/${legacy_keyname}"
-  $keyname = 'RPM-GPG-KEY-puppet'
-  $gpg_path = "/etc/pki/rpm-gpg/${keyname}"
+  $legacy_keyname = 'GPG-KEY-puppetlabs'
+  $legacy_gpg_path = "/etc/pki/rpm-gpg/RPM-${legacy_keyname}"
```

Comment:
I'm preserving the path here so that we don't write another copy for people who are already using this module.